pyproject.toml

```
@@ -3,13 +3,76 @@
 # Note that these settings are not respected with pre-commit run --all-files
 # In that case add configurations to the .pre-commit-config.yaml file.
 
-[project]
+[tool.poetry]
 name = "serve"
-version = "0.0.0"
+version = "0.7.0"
 description = "SciLifeLab Serve is a platform offering machine learning model serving, app hosting (Shiny, Streamlit, Dash, etc.), and other tools to life science researchers affiliated with a Swedish research institute."
-requires-python = "=3.8"
+license = "Apache 2.0"
+maintainers = ["Team Whale <serve@scilifelab.se>",]
+authors = ["Team Whale <serve@scilifelab.se>",]
+homepage = "https://serve.scilifelab.se"
+repository = "https://github.com/ScilifelabDataCentre/stackn"
 keywords = ["machine learning", "life science", "research", "django", "python"]
 
+[tool.poetry.dependencies]
+python = "^3.8.10"
+
+django = "==4.2.7"
+django-celery-beat = "==2.5.0"
+django-compressor = "==4.4"
+django-cors-headers = "==4.2.0"
```

This fixes such kind of error https://stackoverflow.com/a/70371315/8430015
```suggestion
django-cors-headers = "==4.3.1"
```